org.abapgit.adt.backend/src/org/abapgit/adt/backend/internal/RepositoryService.java

```diff
@@ -256,4 +259,75 @@ public URI getURIFromAtomLink(IRepository repository, String relation) {
 		return null;
 	}
 
+	@Override
+	public IAbapGitPullModifiedObjects getModifiedObjects(IProgressMonitor monitor, IRepository currRepository,
+			String user, String password) {
+		//This URI is not a part of the URI's associated with a repository in the backend.
+		//Do/Should we add this to that list in the backend and then fetch the URI from Atom Link, like we do for other URI
+		String uriToModifiedObjectsString = getURIFromAtomLink(currRepository, IRepositoryService.RELATION_PULL).toString()
```

Comment:
Creates a URI in the client (not fetched from backend) of the form
/sap/bc/adt/abapgit/repos/000000000133/pull/modifiedobjects to get the modified objects.

Reply:
Added the atom link in backend and thus fetching the URI from the backend.